lib/pages/backdrop.dart

```diff
@@ -44,7 +46,9 @@ class _BackdropState extends State<Backdrop> with TickerProviderStateMixin {
     super.initState();
     _settingsPanelController = AnimationController(
       vsync: this,
-      duration: const Duration(milliseconds: 200),
+      duration: widget.isDesktop
+          ? const Duration(milliseconds: 200)
```

Comment:
Should these be added to the constants file?

Reply:
Good idea, done